src/VisualStudio/TestUtilities2/ProjectSystemShim/Framework/TestEnvironment.vb

```diff
@@ -74,7 +74,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim.Fr
         <PartNotDiscoverable>
         <Export(GetType(VisualStudioWorkspace))>
         <Export(GetType(VisualStudioWorkspaceImpl))>
-        Private Class MockVisualStudioWorkspace
+        Friend Class MockVisualStudioWorkspace
```

Comment:
Why did these switch to Friend? The implementation should be isolated to these project system tests.

Reply:
This change is now removed from this pull request.